src/world.rs

```diff
@@ -135,6 +135,31 @@ impl World {
         self.spawn_inner(handle, components);
     }
 
+    /// Spawn every `entity` recorded with their components
+    ///
+    /// Useful for recording and spawning entities at some point in the future
+    ///
+    /// # Example
+    /// ```
+    /// # use hecs::*;
+    /// let mut world = World::new();
+    /// let a = world.reserve_entity();
+    /// let mut recorder = EntityRecorder::new();
+    /// recorder.record_entity(a, (false,0.0));
+    /// world.spawn_recorded(&mut recorder);
+    /// assert!(world.contains(a));
+    /// ```
+    pub fn spawn_recorded(&mut self, recorder: &mut EntityRecorder) {
```

Comment:
This should be a method on `EntityRecorder` that takes `&mut World`, perhaps named `run_on`. That way `ReadyRecorder` and various other details of `EntityRecorder` don't need to be exposed, and `World`'s API is unaffected.